test/test_dtype_alu.py

```diff
@@ -60,24 +64,28 @@ class ht:
   bool = strat.booleans()
 
 def universal_test(a, b, dtype, op):
-  if not isinstance(op, tuple): op = (op, op)
+  if not isinstance(op, tuple): op = (op, op, op)
```

Comment:
don't really like this change, i think just using torch for bfloat16 is simpler

Reply:
Ok, will refactor